crates/models/llama/src/lib.rs

```diff
@@ -447,3 +486,13 @@ struct Layer {
     w2: ggml::Tensor,
     w3: ggml::Tensor,
 }
+
+/// Available Llama models
+enum LlamaModelVersion {
+    Model3b,
+    Model7b,
+    Model13b,
+    Model30b,
+    Model65b,
+    Model70b,
+}
```

Comment:
Version hints at the actual llama model version (1 or 2), maybe rename it into `LlamaModelType`?

Reply:
That's true, I changed this in [a16caba] (https://github.com/rustformers/llm/pull/409/commits/a16cabaf5b9b5c1711fe4689e3c00f492524f4e7)